images/spz/spz/templates/internal/registrations.html

```diff
@@ -22,4 +22,16 @@ <h3 class="ui dividing header">Import</h3>
         <button type="submit" class="fluid ui positive button">Einlesen</button>
     </form>
 </div>
+
+<div class="row">
+    <h3 class="ui dividing header">Matrikelnummer/Kürzel-Überprüfung</h3>
+    <form class="ui form" method="post" action="{{ url_for('registrations_verify') }}" enctype="multipart/form-data">
+        {{ csrf_field() }}
+        {{ render_input(form.tag) }}
+        <div class="ui message">
```

Comment:
I would rather pass `tag_exists` as a template parameter (or smth similar) and generate the message on-the-fly within the template. This way, you could also select the message type (e.g. a green message for "OK" and a red/yellow one for "does not exist).

Another thing to keep the emitted HTML as clean as possible is to avoid the `message` div when no message is passed at all (or when `tag_exists` is `None` when applying the clean design described above). You can use control flow statements within the template, see [the official Jinja2 docs](http://jinja.pocoo.org/docs/dev/templates/#if).

In general, all sentences / texts should be pushed down to the template as good as possible. Flash messages are an exception to avoid over-engineering. Same goes for our log system (otherwise you would have to equip every log message with some kind of enum and more metadata and generate the actual text within the template).